apps/server/src/modules/tldraw/tldraw.module.ts

```diff
@@ -3,14 +3,14 @@ import { ConfigModule } from '@nestjs/config';
 import { createConfigModuleOptions, DB_PASSWORD, DB_USERNAME } from '@src/config';
 import { LoggerModule } from '@src/core/logger';
 import { MikroOrmModule, MikroOrmModuleSyncOptions } from '@mikro-orm/nestjs';
-import { AuthenticationModule } from '@src/modules/authentication/authentication.module';
 import { Dictionary, IPrimaryKey } from '@mikro-orm/core';
 import { CoreModule } from '@src/core';
 import { config, TLDRAW_DB_URL } from './config';
 import { TldrawDrawing } from './entities';
 import { TldrawController } from './controller';
 import { TldrawService } from './service';
-import { TldrawBoardRepo, TldrawRepo, YMongodb } from './repo';
+import { TldrawRepo } from './repo';
+import { XApiKeyStrategy } from '../authentication/strategy/x-api-key.strategy';
```

Comment:
ok, so you want to use only this instead of the loading the authentication module
but normally it's not ok to include a file from another module with relative path like this

after doing some testing, looks like this is exactly where the problems happens. You can keep the file storage module, but do not import the authentication module. Very strange

Reply:
Yeah, Its caused by N21-1766 ticket, cap team change the way the dependencies are injected in other modules. This is just workaround for that, not a propper solution

Reply:
that was clear, however, seems you don't need to remove file storage module
only using XApiKeyStrategy instead of importing AuthenticationModule seems to do the trick

Reply:
moreover, I don't get how the AuthGuard really works, and how this XApiKeyStrategy is called. 
What is missing is a test (api / controller test) when user is not valid (to see a 403)

Reply:
we don't need file storage here as we do not delete files here anymore.

Reply:
that might be so, but tldraw is already in production and here we are only trying to fix that thing which breaks it in the new release.